src/content/docs/cheerpj3/11-guides/File-System-support.md

```diff
@@ -2,64 +2,96 @@
 title: File System support
 ---
 
-CheerpJ provides full filesystem support for Java applications running with CheerpJ.
-
-Read only and read/write filesystems are exposed in Java and can be used to read, write and manipulate files as normally when running on a JVM.
-
-**Note**: CheerpJ provides access to a virtualized filesystem, which does not correspond to the local computer. Accessing local files from the browser is forbidden for security reasons.
+CheerpJ provides filesystem support to accommodate different Java application needs. On this page you will find valuable information on how to manage your application data and work with the different filesystem features that CheerpJ provides.
 
 ## File Systems in CheerpJ
 
-CheerpJ implements three main filesystem concepts:
-
-1. A read-only, HTTP-based filesystem
-2. A read/write, IndexedDB-based, persistent filesystem
-3. A read-only, memory based filesystem
-
-CheerpJ filesystems are implemented as UNIX-style virtual filesystems with multiple mount points. The default mount points are defined as follows:
+CheerpJ filesystems are implemented as UNIX-style virtual filesystems with multiple mount points. Brief explanations are found in the table below.
 
 | Mount     | Description                                                                                               |
 | --------- | --------------------------------------------------------------------------------------------------------- |
-| `/app/`   | An HTTP-based read-only filesystem, used to access JARs and data from your local server                   |
+| `/app/`   | An HTTP-based read-only filesystem used to access JARs and data from your local server                    |
 | `/files/` | An IndexedDB-based, persistent read-write file system                                                     |
-| `/lt/`    | Another HTTP-based read-only filesystem, pointing to the CheerpJ runtime                                  |
 | `/str/`   | A read-only filesystem to easily share JavaScript Strings or binary data (an `Uint8Array`) with Java code |
 
+![](/cheerpj2/assets/filesystem.png)
+
+> [!warning]Important
+> CheerpJ provides access to a virtualized filesystem, which does not correspond to the local computer. Accessing local files from the browser is forbidden due to security reasons.
+
 ## `/app/` mount point
 
 The /app/ mount point corresponds to a virtual read-only, HTTP-based filesystem. `/app/` is used to access JAR files and data from your local server.
 
-The `/app/` directory is virtual, and only exists inside of CheerpJ. It is required to distinguish files from the local server from runtime files and files stored in the browser database.
+This mount point is virtual and only exists inside of CheerpJ. It is required to distinguish between local server, runtime files and files stored in the browser database.
```

Comment:
I am not sure about the purpose of this sentence. All mount points are virtual and only exists inside of CheerpJ

Reply:
I would remove this para entirely